src/Views/Form/Templates/Sequoia/Sequoia.php

```diff
@@ -97,16 +97,27 @@ public function loadScripts() {
 				border: 1px solid {$primaryColor} !important;
 			}
 			.checkmark {
-				border-color: {$primaryColor}!important;
-				color: {$primaryColor} !important;
+				border-color: {$primaryColor} !important;
+				color: {$primaryColor}!important;
 			}
 			input[type='radio'] + label::after {
 				background: {$primaryColor} !important;
 			}
 		";
 		wp_add_inline_style( 'give-sequoia-template-css', $dynamicCss );
 
-		$rawColor              = trim( $primaryColor, '#' );
+		$rawColor            = trim( $primaryColor, '#' );
+		$recurringDynamicCss = "
```

Comment:
@henryholtgeerts use i think using `sprintf` is a better choice when replace when add dynamic value to string.
Let me know if you dis agree

ref: https://stackoverflow.com/questions/7147305/performance-of-variable-expansion-vs-sprintf-in-php

Reply:
Here sprintf is less clean than string interpolation because of the data-url used to setup the checkmark's background. It is more readable because we don't have to worry about escaping %s that appear within the svg markup.

Reply:
@henryholtgeerts I am talking about `$primaryColor` because it is dynamic. No need to use do it for data URL because that will remain the same.

Reply:
Ahh understood. I'll make that change.

Reply:
I tend to reach for basic string interpolation if I just want a direct injection. `sprintf` is great as it provides very lightweight formatting. So if it's important to assert the types being injected, then use `srpintf`.

Reply:
Done, ref: https://github.com/impress-org/givewp/commit/2b93fd4258873353e813a54585c5562621dc6015